extensions/wikia/CreateNewWiki/maintenance/importStarter.php

```diff
@@ -158,7 +158,14 @@ public function execute() {
 		/* @var Language $wgContLang */
 		global $wgContLang, $wgCityId, $wgDBname;
 		$language = $wgContLang->getCode();
-		$starter = Wikia\CreateNewWiki\Starters::getStarterByLanguage( $language );
+		$wikiVertical = (int) getopt( '', [ 'vertical:' ] )['vertical'];
+
+		// TODO Clean up in XW-2393
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)